test/wpool_SUITE.erl

```diff
@@ -104,12 +119,13 @@ too_much_overrun(_Config) ->
   TCPid ! {check, Worker, TaskId, 100}, % tiny runtime, to check
 
   ct:comment("Nothing happens..."),
-  receive
-  after 1000 -> ok
-  end,
+  _ = case get_messages(1000) of
```

Comment:
@HernanRivasAcosta you can always configure elvis (through elvis.config) appropriately.